src/codeu/chat/common/Bot.java

```diff
@@ -0,0 +1,13 @@
+package codeu.chat.common;
+
+public abstract class Bot {
+
+    abstract String reactTo(String command, String[] args);
```

Comment:
What happens if a bot just wants to get a dumb string message instead of a command and arguments?

I would have maybe expected this function to just be `reactTo(String sender, String message)`, and subclasses could split the `message` into commands if they wanted (maybe with a handy util funciton in this class). If doing it this way was easier, that's fine!

Also, does a bot know who sent a message? If not, will that get messy in busy chat rooms?